admin-core/src/main/resources/application.conf

```diff
@@ -98,20 +92,12 @@ drill {
             roleMapping {
             }
         }
-        simpleAuth {
+        simple {
             enabled = true
             enabled = ${?DRILL_SIMPLE_AUTH_ENABLED}
             signUpEnabled = true
-        }
-        password {
-            minLength = 6
-            minLength = ${?DRILL_PASSWORD_MIN_LENGTH}
-            mustContainUppercase = false
-            mustContainUppercase = ${?DRILL_PASSWORD_CONTAIN_UPPERCASE}
-            mustContainLowercase = false
-            mustContainLowercase = ${?DRILL_PASSWORD_CONTAIN_LOWERCASE}
-            mustContainDigit = false
-            mustContainDigit = ${?DRILL_PASSWORD_CONTAIN_DIGIT}
+            passwordStrength {
```

Comment:
Reason for removing defaults? It will require us/user to dig through module implementation to set these. I suggest to set all params to "false"/unset values, but keep the setting names

Reply:
We already don't have all settings here -> this is _not_ a single source of truth for all settings
Add exhaustive configuration description to website docs